### PR TITLE
169842519 Update README file

### PR DESCRIPTION
**Pull request by victkarangwa**
### What does this PR do?
Update README file 
#### Description of Task to be completed?
This PR update README file with guidance for running and testing the application
#### How should this be manually tested?
- Go to `ch-update-readme-file-169842519` branch
- Scroll down to the README file
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169842519](https://www.pivotaltracker.com/story/show/169842519)
#### Screenshots
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by william000000 — APPROVED** [review]

**Review by william000000 — COMMENTED** [review]
Good Job @victorkarangwa4

**Review by ivymwende — APPROVED** [review]
Great Job @victorkarangwa4 !

**Review by NiyoEric — APPROVED** [review]
You wrote a good readme @victorkarangwa4! we will update it as we along.

**Review by hezronkimutai — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,8 +3,135 @@
...
+| [/api/v1/auth/verify-email/[token]](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/verify-email/) | GET |
+
+- #### Resend email endpoint
+
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/:id/resend-email](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/:id/resend-email) | GET |
+
+# Author :computer:
+[Andela Kigali](https://www.andela.com) Team-odd
```
Comment (ceelogre) [inline README.md]: This link is irrelevant as it's for Andela global.

File: README.md [hunk 2]
```diff
@@ -3,8 +3,135 @@
...
+
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/verify-email/[token]](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/verify-email/) | GET |
+
+- #### Resend email endpoint
+
+| Endpoint | Request Method |
+| ---------- |----------- |
+| [/api/v1/auth/:id/resend-email](https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/:id/resend-email) | GET |
```
Comment (ceelogre) [inline README.md]: This should be put in Documentation not here. Think about it,  as you progress you'll create new endpoints, will you keep adding them here?

File: README.md [hunk 3]
```diff
@@ -3,8 +3,135 @@
...
+
+### A. Clone the Application
+
+1. Open your terminal 
+
+2. Run `clone https://github.com/andela/team-odd-bn-backend.git` OR [Download](https://github.com/andela/team-odd-bn-backend/archive/develop.zip) the project
+
+### B. Setting up the environment
+
+1. **Create** a `.env` file to put in all environmental variables
```
Comment (ceelogre) [inline README.md]: The project already has `.env.example` so as part of this pull request, you're supposed to rename it to `.env` then all users have to do is add the variables, not create new files.

**Review by ceelogre — CHANGES_REQUESTED** [review]

**Review by julietezekwe — APPROVED** [review]

**Pull request merged**